### DISCUSSION TITLE
There are outstanding issues with your GitHub account

### DISCUSSION THREAD

**Discussion by sekharmalla** [community/community#51341] [GitHub Education]
There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.

image.png

However When I went there to re-verify I got this.
![image](https://user-images.githubusercontent.com/43902003/228584712-9bb4145e-f8b0-44c2-aaba-9e4c15d56a4d.png)

![image](https://user-images.githubusercontent.com/43902003/228580212-4c54caaf-ea3a-4a26-bf11-bb7e315333f6.png)

**Comment by samus-aran** [comment]
Hi @sekharmalla 👋 

This message can be an indication of many things that only the Support team can answer and resolve. So please do contact [Account or billing issues](https://support.github.com/request) Support and they will explain the details of this message.

**Reply by sekharmalla** [reply]
Hi @samus-aran I understand but I was sent from Account or Billing issues Support to this community. It's been nearly a month these people just making me move from here to there and there to here.
